packages/swingset-theme-hashicorp/src/components/side-nav/toggle-button.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconChevronRight24 } from '@hashicorp/flight-icons/svg-react/chevron-right-24'
+import { cx } from 'class-variance-authority'
+
+type ToggleButtonProps = {
+ toggle: () => void
+ isOpen: boolean
+}
+
+export function ToggleButton({ toggle, isOpen }: ToggleButtonProps) {
+ return (
+  <button className="ss-fixed ss-left-72 ss-p-2 ss-top-6" onClick={toggle}>
+   <div
+    className={cx(
+     'ss-bg-surface-faint ss-border-faint ss-border-2 ss-rounded-full ss-transition-transform ss-shadow-md ss-p-1 ss-bg-opacity-60 ss-backdrop-blur-sm',
+     !isOpen && 'ss--rotate-180'
+    )}
+   >
+    <IconChevronRight24 />
```

Comment:
What do you think about using `ChevronLeft` as the default? That direction feels more intuitive to me

Reply:
+1 here:

When open, the chevron should point left to denote the intent of the button click.

Similarly, when closed, the chevron should point right because since the intent is to open the menu.

Current:
<img width="415" alt="CleanShot 2023-06-23 at 13 02 02@2x" src="https://github.com/hashicorp/swingset/assets/26389321/32c2d98a-eac6-42c6-b467-96f206c4e53f">